user-interface/src/main/java/life/qbic/datamanager/views/projects/overview/ProjectOverviewPage.java

```diff
@@ -45,19 +45,29 @@ public class ProjectOverviewPage extends Div {
   private final FinanceService financeService;
   private final OntologyTermInformationService ontologyTermInformationService;
   private final AddExperimentToProjectService addExperimentToProjectService;
+  private final ContactRepository contactRepository;
 
-  public ProjectOverviewPage(@Autowired ProjectCollectionComponent projectCollectionComponent,
+  public ProjectOverviewPage(ProjectCollectionComponent projectCollectionComponent,
       ProjectCreationService projectCreationService, FinanceService financeService,
       OntologyTermInformationService ontologyTermInformationService,
-      AddExperimentToProjectService addExperimentToProjectService) {
+      AddExperimentToProjectService addExperimentToProjectService,
+      ContactRepository contactRepository) {
     this.projectCollectionComponent = projectCollectionComponent;
     this.projectCreationService = projectCreationService;
     this.financeService = financeService;
     this.ontologyTermInformationService = ontologyTermInformationService;
     this.addExperimentToProjectService = addExperimentToProjectService;
-    layoutPage();
-    configurePage();
-    stylePage();
+    this.contactRepository = contactRepository;
+    add(this.projectCollectionComponent);
+    this.projectCollectionComponent.addCreateClickedListener(projectCreationClickedEvent -> {
+      AddProjectDialog addProjectDialog = new AddProjectDialog(this.financeService,
+          this.ontologyTermInformationService, this.contactRepository);
+      addProjectDialog.addConfirmListener(this::createProject);
+      addProjectDialog.addCancelListener(it -> it.getSource().close());
+      addProjectDialog.open();
+    });
+    this.setWidthFull();
+    this.setHeightFull();
```

Comment:
NIT: These should be moved to CSS

Reply:
aggreed. This is an artifact of inlining the methods.